packages/nimble-tokens/source/styledictionary/properties/colors.json

```diff
@@ -176,6 +176,30 @@
     },
     "NI Sea Dark 2": {
       "value": "#00685Fff"
+    },
```

Comment:
Some questions about these colors:

1. Where do the names come from? Do they match values that @NIbokeefe is using in a Visual Design document?
2. What is the naming scheme? What do the numbers indicate? Do they mean the same thing as the numbers on existing tokens?
3. It looks like our pattern has been to use RGBA values rather than RGB values (i.e. to include "ff" for the alpha value at the end). Is there a reason not to do this for these colors?

Depending on the answers to these questions we should think about the naming convention. For example, if it's necessary to express these in RGB do we want to use "Opaque" to signify that? Should the number go at the end or next to the thing it's modifying? Should we sort them alphabetically / keep similar colors together / keep all the opaque colors together?

Reply:
Thanks for the explanation and the changes. The new color tokens seem good: they match existing patterns in that they are sourced from VxD, follow consistent naming, and specify values in RGBA.

I didn't completely follow your explanation about the remaining issue. I'll summarize my understanding; please clarify if I misinterpreted:
1. in the gif, the background of the card gets darker while it is being resized and the original size remains visible, both of which are undesirable
2. both are caused by using base tokens with transparency instead of opaque tokens
3. these new tokens are opaque and we can use them to generate theme-aware tokens with transparency to solve this problem

If that's accurate then I think the current approach is good.

Some more background in case that's not accurate: sometimes we have to compute opaque values whose values are equal to what would result if a given foreground color was made partially transparent and displayed over a given background color. We generally avoid doing this because we don't always know what the background color will be (for example the Color theme can be used with any background). But we've done it in a few places when necessary and we could talk about doing it here if needed.

Reply:
> both are caused by using base tokens with transparency instead of opaque tokens

These issues arise because theme-aware tokens are generated by converting base tokens to hex using the `hexToRgbaCssColor` function, which returns the color in `rgba` format. This conversion causes problems, especially when these theme-aware tokens are utilized. 

![image](https://github.com/user-attachments/assets/9c93781a-87c8-499e-8ef6-9189385439a4)

To address this, I have added a few base tokens with the actual hex values. This approach helps avoid the issues that occur when the desired tokens are generated with transparency using `hexToRgbaCssColor` as `rgba`.